app/components/missing_details/style.scss

```diff
@@ -0,0 +1,11 @@
+@import "../base.scss";
+
+.govuk-inset-text {
+border-left: 5px solid #1d70b8;
+}
+
+.govuk-heading-s {
+  color: #1d70b8;
+  line-height: 0;
+  margin-bottom: 10px
+}
```

Comment:
@avinhurry can we review these? we shouldn't redefine govuk styles. I think we can probably make do with a modifier on the existing inset text component.